shared/src/components/enroll/message-list.cjsx

```diff
@@ -0,0 +1,34 @@
+React = require 'react'
+_ = require 'underscore'
+
+MessageList = React.createClass
+
+  propTypes:
+    messages: React.PropTypes.array.isRequired
+    alertType: React.PropTypes.string
+    messagesType: React.PropTypes.string
+    forceBullets: React.PropTypes.bool
+
+  getDefaultProps: ->
+    alertType: "danger"
+    messagesType: "errors"
+    forceBullets: false
+
+  getUlClassName: ->
+    if @props.messages.length == 1 && !@props.forceBullets
+      "#{@props.messagesType} list-unstyled"
+    else
+      @props.messagesType
+
```

Comment:
This could be replaced by the [classnames utility](https://github.com/JedWatson/classnames) (after adding `classnames = require 'classnames'`) and moved to render as:
```coffee
classnames = classnames('message-list', @props.messagesType, 
  {'list-unstyled': @props.messages.length == 1 && !@props.forceBullets}
)
```

Note that the above also adds a "message-list" class, which is a nice thing to do for components so they can be targeted for custom styling.